cmd/gendocs/gen_kubectl_docs.go

```diff
@@ -45,7 +45,6 @@ func main() {
 	// Set environment variables used by kubectl so the output is consistent,
 	// regardless of where we run.
 	os.Setenv("HOME", "/home/username")
-	// TODO os.Stdin should really be something like ioutil.Discard, but a Reader
-	kubectl := cmd.NewKubectlCommand(os.Stdin, ioutil.Discard, ioutil.Discard)
+	kubectl := cmd.NewKubectlCommand(io.Reader, io.Discard, io.Discard)
```

Comment:
I don't think io.Reader can substitute os.Stdin here. Reader is an interface and can't be instantiated on its own - os.Stdin is a constant that implements the Reader interface.

Reply:
I will change it back! Or do you have another suggestion (based on the TODO on line 48?